diff --git a/libs/hbb_common/src/config.rs b/libs/hbb_common/src/config.rs
index a053889..6c20154 100644
--- a/libs/hbb_common/src/config.rs
+++ b/libs/hbb_common/src/config.rs
@@ -21,14 +21,14 @@ pub const DEFAULT_VIDEO_BITRATE: u32 = 2000;
 pub const DEFAULT_AUDIO_BITRATE: u32 = 128;
 pub const RENDEZVOUS_SERVERS: &[&str] = &[
-    "rs-ny.rustdesk.com",
-    "rs-sg.rustdesk.com",
-    "rs-de.rustdesk.com",
+    "rustdesk.newdebao.com:21116",
 ];
 pub const RELAY_SERVERS: &[&str] = &[
-    "rr-ny.rustdesk.com",
-    "rr-sg.rustdesk.com",
-    "rr-de.rustdesk.com",
+    "rustdesk.newdebao.com:21117",
 ];
-pub const RS_PUB_KEY: &str = "官方公钥";
+pub const RS_PUB_KEY: &str = "zp4ev7m8mR2hqMB4hjxxH9GbroH8tzW8VbPYW4a5ufo=";
 
 pub const UPDATE_URL: &str = "";
 
@@ -221,6 +221,16 @@ pub static ref HARD_SETTINGS: RwLock<HashMap<String, String>> = {
     let mut map = HashMap::new();
     map.insert("allow-auto-update".to_string(), "N".to_string());
     map.insert("hide-server-settings".to_string(), "Y".to_string());
+    map.insert("telemetry".to_string(), "N".to_string());
+    map.insert("allow-file-transfer".to_string(), "Y".to_string());
+    map.insert("allow-clipboard".to_string(), "Y".to_string());
+    map.insert("allow-keyboard".to_string(), "Y".to_string());
+    map.insert("allow-mouse".to_string(), "Y".to_string());
+    map.insert("allow-restart".to_string(), "Y".to_string());
+    map.insert("allow-recording".to_string(), "Y".to_string());
+    map.insert("view-style".to_string(), "full".to_string());
+    map.insert("auto-accept".to_string(), "Y".to_string());
     RwLock::new(map)
 };
